interactive_ai/services/director/app/communication/exceptions.py

```diff
@@ -586,7 +586,8 @@ class ProjectConfigurationNotFoundException(GetiBaseException):
 
     def __init__(self, project_id: ID, task_id: ID | None = None) -> None:
         message = (
-            f"The requested project configuration could not be found. Project ID: `{project_id}`, task ID: `{task_id}`"
+            f"The requested project configuration could not be found. Project ID: `{project_id}`"
+            + f", task ID: `{task_id}`"
```

Comment:
```suggestion
            f"The requested project configuration could not be found. Project ID: `{project_id}`, "
            f"task ID: `{task_id}`"
```
FYI the `+` is unnecessary, strings are concatenated after line break

Reply:
In this particular case `+` is necessary because we want to include the task ID only when it's available. If we don't use `+` the message would be `""` since we have a `if task_id else ""` condition